raster_download/.gitignore

```diff
@@ -1,4 +0,0 @@
-target/
```

Comment:
Why are these removed from `.gitignore`? `target` for example is created when you run `mvn package`. So no need to check it in.